app/scripts/controllers/projects.js

```diff
@@ -16,13 +16,37 @@ angular.module('openshiftConsole')
     $scope.showGetStarted = false;
     $scope.canCreate = undefined;
 
+    var sortProjects = function() {
+      var sortID = _.get($scope, 'sortConfig.currentField.id', 'metadata.name');
+
+      $scope.projects = _.sortByOrder($scope.projects, function(project) {
+        return _.get(project, sortID).toLowerCase() || _.get(project, 'metadata.name').toLowerCase();
```

Comment:
It looks like `_.sortByOrder` should take an array of functions, not a single function.